src/aws/autoscaling.ts

```diff
@@ -38,12 +38,3 @@ export function describeAsg(asgName: string): Promise<AutoScaling.Types.AutoScal
     const params = { AutoScalingGroupNames: [ asgName ] };
     return retry(() => awsAutoscaling.describeAutoScalingGroups(params).promise(), `describing ASG ${asgName}`, 5)
 }
-
-export function getDesiredCapacity(asgInfo: AutoScalingGroupsType): number {
```

Comment:
I've removed this because I think it's clearer that we're returning a single ASG and getting the desired capacity with the `singleASG` function. `singleASG` is only used by the autoScalingGroupCheck and it's not making a call to AWS like the  other methods here, so I've left it in the autoScalingGroupCheck file.